.remarkrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "plugins": [
+    "remark-preset-prettier",
+    "remark-frontmatter",
+    "remark-preset-lint-node",
+    ["remark-gfm", false],
```

Comment:
TIL you can turn off an inherited plugin like this

Reply:
Same, only learned it now!